lab-max/app/component/gallery/edit-gallery/_edit-gallery.scss

```diff
@@ -0,0 +1,8 @@
+@import "theme";
+
+.update-gallery-container {
+
```

Comment:
@meeksfred no need for a line break here - simply because this is such small file